package.json

```diff
@@ -51,6 +52,7 @@
     "case-sensitive-paths-webpack-plugin": "2.1.1",
     "chalk": "2.3.0",
     "color": "3.0.0",
+    "cross-env": "^5.2.0",
```

Comment:
Nit: can you please remove the carets from the new dependencies `cross-env` and `yarn`? Just to be consistent / allow for safe rebuilding of yarn.lock